src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/resource.js

```diff
@@ -0,0 +1,26 @@
+window.addEventListener("DOMContentLoaded", () => {
+    hljs.initHighlightingOnLoad();
+
+    var showVersionDiffsJs = true;
+
+    function toggleShowHideVersionDiffsJs(button, hideText, showText) {
+        showVersionDiffsJs = !showVersionDiffsJs;
+        if (showVersionDiffsJs === true) {
+            document.getElementById("tbody_versionDiffsShown").style.display = '';
+            document.getElementById("tbody_versionDiffsHidden").style.display = 'none';
+
+            button.value = hideText;
+        } else {
+            document.getElementById("tbody_versionDiffsShown").style.display = 'none';
+            document.getElementById("tbody_versionDiffsHidden").style.display = '';
+
+            button.value = showText;
+        }
+    }
+
+    document.querySelector("#showHideVersionDiffsJsButton").addEventListener("click", (event) => {
+        const { hideText, showText } = event.target.dataset;
```

Comment:
This is well supported according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#browser_compatibility.
I could switch to the old way of doing that though if that's too uncommon.